test/runtime/configMatters/comm/AtomicsAPI.chpl

```diff
@@ -37,6 +41,12 @@ proc testAtomicT(a, ref i, ref b, type basetype) {
 
   writeln("Testing 'atomic " + basetype:string + "':");
                                                     writea ("init    ", a);
+                if isArray(a) == false {
+                  var initval : a.type = 1;         writeai("init=   ", a, initval);
+                  var initval2 : a.type = a;        writeai("init=(a)", a, initval2);
+                  var initval3 = a;                 writeai("init=(a)", a, initval3);
```

Comment:
If you're still making changes, maybe change these to initval2 = initval, and initval3 = initval, just since right now the default and `a` have the same value of `0` (so we wouldn't notice if say `proc init=(other)` was default initializing instead of taking `other's value